extension/goext/schemas.go

```diff
@@ -20,6 +20,9 @@ import (
 	"errors"
 )
 
+// SchemaID is a type for schema ID
+type SchemaID string
```

Comment:
What's the benefit of creating new type in this case?

Reply:
type safety